tests/UserAgentsTest/FullTest.php

```diff
@@ -207,6 +207,10 @@ public function testUserAgents(string $userAgent, array $expectedProperties) : v
                 continue;
             }
 
+            if (self::$propertyHolder->isDeprecatedProperty($propName, self::$writer)) {
+                continue;
```

Comment:
Should this maybe throw a notice or something so that we know that the test contains deprecated properties and should be updated?